### ISSUE TITLE
transmission downloads to a blackhole if the incomplete dir isn't writeable

### ISSUE THREAD

**Issue by SeleniumCake** [transmission/transmission#5699]
### What is the issue?

If I go chmod 044 ./incomplete_dir/ then transmission will continue downloading the torrent without saving it anywhere.  An error is displayed if the torrent is paused, but it can be resumed and starts downloading into nothingness.

I've tested it on macOS 11.7 and Synology DSM 6.2.4-25556 Update 5

### Which application of Transmission?

None

### Which version of Transmission?

4.03

**Comment by ckerr** [comment]
...I feel like showing an error and pausing the torrent is a pretty reasonable response to this?

**Comment by SeleniumCake** [comment]
> ...I feel like showing an error and pausing the torrent is a pretty reasonable response to this?

Yes, showing an error and pausing the torrent would be ideal.  But that's not the current behaviour, which is the problem.

**Comment by ckerr** [comment]
When I `chmod 044` the incomplete directory and start downloading a torrent, the torrent downloads for a bit until it comes time to write to disk. At that point, Transmission pauses the torrent and sets the torrent's error state to

> Error: Couldn't get ${incomplete_dir}/${torrent_file}': Permission denied

So it seems to be working as intended..? Maybe I'm misunderstanding your bug report?

**Comment by ckerr** [comment]
Closing stale issue due to inability to reproduce + lack of actionable info.

If anyone experiences this in >= 4.1.0, please open a new ticket with updated info. Thanks.
